project/src/main/webapp/script.js

```diff
@@ -134,28 +134,40 @@ function createPlaceElement(place) {
   const name = document.createElement('li');
   name.innerText = place.name;
   placeElement.appendChild(name);
-  placeElement.appendChild(document.createElement('br'));
   // Add link to website
   if (place.websiteUrl) {
-    const websiteLink = document.createElement('a');
-    websiteLink.href = place.websiteUrl;
-    websiteLink.title = place.websiteUrl;
-    websiteLink.innerText = 'Restaurant\'s website';
-    placeElement.appendChild(websiteLink);
-  } else {
-    const noSite = document.createTextNode('We don\'t have a link to the restaurant\'s website.');
-    placeElement.appendChild(noSite);
+    addLinkToPlaceElement(placeElement, place.websiteUrl, 'Website');
+  }
+  if (place.googleUrl) {
+    addLinkToPlaceElement(placeElement, place.googleUrl, 'Google Maps link');
+  }
+  if (!place.websiteUrl && !place.googleUrl) {
+    placeElement.appendChild(document.createElement('br'));
+    placeElement.appendChild(document.createTextNode(
+        'We don\'t have a link to the restaurant\'s website.'));
   }
   placeElement.appendChild(document.createElement('br'));
   // Add phone number
   if (place.phone) {
-    const phone = document.createTextNode('Phone number:' + place.phone);
+    const phone = document.createTextNode('Phone number: ' + place.phone);
     placeElement.appendChild(phone);
     placeElement.appendChild(document.createElement('br'));
   }
   return placeElement;
 }
 
+/**
+ * Returns a link that represents the given url and shows the given text.
+ */
+function addLinkToPlaceElement(placeElement, url, linkText) {
+  placeElement.appendChild(document.createElement('br'));
+  const link = document.createElement('a');
+  link.href = url;
+  link.title = url;
```

Comment:
We should check (when we deal with front end) that the urls aren't too long and ugly to be supplied as a title (displayed when hovering). This is especially for the google pages urls which tend to be long. I don't care that we leave it for now but just keep in mind this might won't look good and then we can change the title to a description (like "go to link") or just not supply it.

Reply:
That's a good point. I don't mind not adding the title at all, or using a general term like "go to the website". Do you have a preference?

Reply:
I think we can give up on the title

Reply:
done